variables.tf

```diff
@@ -105,4 +105,17 @@ variable "sysdig_metrics_filter" {
   }
 }
 
+variable "sysdig_agent_tags" {
+  type        = list(string)
+  description = "array of tags to group the host logs pushed by the sysdig agent"
```

Comment:
sysdig deals with metrics, not logs

Reply:
I guess it would be just replacing logs with metrics?